### PR TITLE
Add some JUnit tests for BusData class

### LINKED ISSUE #34: As a freshman/ anyone new to NUS, I want to know the buses available at specific bus stops...
... so that I can better plan my trip around the campus in advance

### LINKED ISSUE #38: JUnit tests for BusData class busAtStop method
test for busAtStop method in BusData class

### PR DESCRIPTION
**Pull request by EthanWong2212**
JUnit tests for busAtStop method in BusData class
Fix #38

### CODE REVIEW

File: src/main/java/seedu/duke/BusStops.java [hunk 1]
```diff
@@ -35,6 +38,15 @@ public String getName() {
         return name;
     }
 
+    public static String formatName(String input) throws CustomException {
+        for (BusStops info: EnumSet.allOf(BusStops.class)) {
+            if (info.getName().equalsIgnoreCase(input)) {
```
Comment (yuxinwu99) [inline src/main/java/seedu/duke/BusStops.java]:
```suggestion
            if (info.name.equalsIgnoreCase(input)) {
```

Comment (yuxinwu99) [reply src/main/java/seedu/duke/BusStops.java]: usage of getName function is redundant.

File: src/test/java/seedu/duke/BusDataTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @Test
+    void busAtStop_validStopAllCaps_expectAA1AA2() {
+        String input = "UNIVERSITY TOWN";
+        ArrayList<Bus> busList = BusData.busAtStop(input);
+        assertEquals(2, busList.size());
+        assertEquals("AA1", busList.get(0).getBusNumber());
+        assertEquals("AA2", busList.get(1).getBusNumber());
+    }
```
Comment (yuxinwu99) [inline src/test/java/seedu/duke/BusDataTest.java:123]: Good test cases for the program

File: src/test/java/seedu/duke/logic/commands/BusCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package seedu.duke.logic.commands;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import seedu.duke.exceptions.CustomException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```
Comment (wamikamalik) [inline src/test/java/seedu/duke/logic/commands/BusCommandTest.java]: can this import be removed?
```suggestion
```

Comment (EthanWong2212) [reply src/test/java/seedu/duke/logic/commands/BusCommandTest.java]: I've removed this file. Wasn't meant to commit it to branch

Comment (wamikamalik) [reply src/test/java/seedu/duke/logic/commands/BusCommandTest.java]: Okie! Yah, I realized it wasn't running!

**Review by wamikamalik — APPROVED** [review]
Looks good to merge!

**Pull request merged**